packages/backend/src/services/rudolfo/rudolfo.model.ts

```diff
@@ -0,0 +1,13 @@
+import NeDB from 'nedb';
+import path from 'path';
+import { Application } from '../../declarations';
+
+export default (app: Application): NeDB<any> => {
+  const dbPath = app.get('nedb');
+  const Model = new NeDB({
+    filename: path.join(dbPath, 'rudolfos.db'),
+    autoload: true,
```

Comment:
you could add  the following line after autoload, for the test to pass alyways
`inMemoryOnly: process.env.NODE_ENV === 'test',`